android/src/main/java/io/cobrowse/reactnative/CobrowseIOModule.java

```diff
@@ -240,4 +253,14 @@ public void overwriteSessionIndicator() {
 
     CobrowseIO.instance().setDelegate(delegatesWithSessionControls);
   }
+
+  @ReactMethod
+  public void addListener(String eventName) {
+    // Keep: Required for RN built in Event Emitter Calls.
+  }
+
+  @ReactMethod
+  public void removeListeners(Integer count) {
+    // Keep: Required for RN built in Event Emitter Calls.
+  }
```

Comment:
Fixes the following warning
![Uploading Screenshot 2023-05-05 at 21.49.28.png…]()